### PR TITLE
Start proxy with ocis server command 

### PR DESCRIPTION
**Pull request by IljaN**
Issues #119, #136

### CODE REVIEW

**Comment by IljaN** [comment]
@refs Could it be a problem that proxy has the --config-file flag in the global flags i.e might clash with ocis --config-file? Or did we decide that global config file will also be json and thus config-files will be merged?

File: pkg/command/proxy.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	}
+}
+
+func configureProxy(cfg *config.Config) *svcconfig.Config {
+	cfg.Proxy.Log.Level = cfg.Log.Level
+	cfg.Proxy.Log.Pretty = cfg.Log.Pretty
+	cfg.Proxy.Log.Color = cfg.Log.Color
+	cfg.Proxy.Tracing.Enabled = false
+	cfg.Proxy.HTTP.Addr = "localhost:9200"
+	cfg.Proxy.HTTP.Root = "/"
```
Comment (butonic) [inline pkg/command/proxy.go]: remove these ... they are the defaults taken from ocis-proxy:
- tracing enabled default is not set (which equals to false): https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L58-L63
- http addr defaults to 0.0.0.0:9200 https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L120
  we can debate if we want to allow requests from everywhere or only from localhost. if we want to only allow localhost wen need to set the http address for all other services. then we should normalize across all ocis-commands
- http root is / by default: https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L127

Comment (butonic) [reply pkg/command/proxy.go]: I vote for locking all services but ocis-proxy to localhost. that way only one service is reachable and ocis can be set up for testing in eg docker.

**Comment by micbar** [comment]
@IljaN @refs How would I start the server now with the default config?

copy the proxy.json to ~/.ocis/config ?

**Comment by IljaN** [comment]
@micbar Your safest bet is to pass the path of the config-file in to PROXY_CONFIG_FILE. Not sure if --config-file or defaults work atm (see my question above)

**Pull request merged**